INSTALLATION.md

```diff
@@ -2,17 +2,36 @@
 
 These instructions should help you get the Cilium Service Mesh-specific images installed in your cluster, with the service mesh features enabled. 
 
-## Install Cilium and Hubble
+## Install Cilium
 
 [Install the Cilium CLI](https://docs.cilium.io/en/v1.11/gettingstarted/k8s-install-default/#install-the-cilium-cli) and run `cilium version` to check that you have version 0.10.2 or above.
 
-Install Cilium with the service mesh builds, and enable Hubble:
+Install Cilium with the service mesh builds. 
+
+* If you want Cilium to co-exist alongside kube-proxy:
+
+```
+cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-config=true --kube-proxy-replacement=partial --config enable-node-port=true
```

Comment:
`enable-node-port` requires Linux kernel 4.16 or newer. Maybe we should have a recommended minimum Linux version for Cilium Service Mesh?

Reply:
I think it requires 4.19.57: https://docs.cilium.io/en/latest/operations/system_requirements/#required-kernel-versions-for-advanced-features.